### PR TITLE
Readme improvement 

### PR DESCRIPTION
**Pull request by lamiazar**

### CODE REVIEW

**Review by chicoxyzzy — APPROVED** [review]
🤩

File: README.md [hunk 1]
```diff
@@ -1,23 +1,46 @@
...
 
-See the [`test/`](test/) for the unit and integration test suite.
+Test all the things! See the [`test/`](test/) for the unit and integration test suite.
 
-### Development
+### Contributing
 
-If you want to contribute to the Socket Runtime project itself, please connect
-with any current project contributor.
+Please stop by [Discord](https://discord.com/invite/YPV32gKCsH) and ask the team about your issue and if someone is already working on it.
```
Comment (mribbons) [inline README.md]: We can think about publishing install.sh and install.ps1 instructions now. These are used for building the CLI and app libs.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Socket Runtime
                                              
-__Write once, run anywhere, connect everyone__ 
+⚡__Write once, run anywhere, connect everyone__ ⚡
```
Comment (mribbons) [inline README.md]: 🔥

**Review by mribbons — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,23 +1,57 @@
...
+  
+  
 
 ### Description
 
-Socket is a client-side runtime for creating native cross-platform software on
-mobile and desktop using HTML, CSS, and JavaScript. It also exposes primitives
-needed for building peer-to-peer and local-first applications, such as Bluetooth,
-UDP and File I/O.
+`Socket Runtime`. provides a range of tools and software utilities to help developers build powerful application experiences using standard web technologies `(HTML, CSS, and JS)`.You can also connect your users, and let them communicate directly, without the cloud or any servers!
```
Comment (jwerle) [inline README.md:15]: cc @heapwolf

**Review by jwerle — APPROVED** [review]
love the structure and layout! @heapwolf may have some final thoughts on copy

File: README.md (hunk 1 above)
Comment (lamiazar) [reply README.md]: >If you want to contribute to the Socket Runtime project itself, please connect
with any current project contributor.

I think we can have a detailed `contribution Guide` in the near future, I liked how many other websites have that. This guide can be a link in `readme` or a `detailed section` in the docs.

File: README.md [hunk 4]
```diff
@@ -1,23 +1,64 @@
...
 
-### Documentation
+Test all the things! See the [`test/`](test/) for the unit and integration test suite.
 
-<!-- [![Socket SDK CI](https://github.com/socketsupply/socket/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/socketsupply/socket/actions/workflows/ci.yml) -->
+### 🙏 Contributing
 
-Please read (and contribute to) the documentation [here](https://sockets.sh).
+Please stop by [Discord](https://discord.com/invite/YPV32gKCsH) and ask the team about your issue and if someone is already working on it.  
+Please connect with any current project contributors : [@heapwolf][0], [@jwerle][1], or [@chicoxyzzy][2] if you want to contribute to the Socket Runtime project itself.
```
Comment (chicoxyzzy) [inline README.md:56]: nit: there's an extra space before colon

File: README.md [hunk 5]
```diff
@@ -1,23 +1,64 @@
...
+
+This is similar to React's `Create React App`, we provides a few basic boilerplates and some strong opinions so you can get coding on a production-quality app as quickly as possible.  
+Please check [create-socket-app Repo](https://github.com/socketsupply/create-socket-app) to get started and to learn more.  
+You can also check our `Examples` in the [Examples Repo](https://github.com/socketsupply/socket-examples).  
+
+
+### 📚 Documentation
+
+The full documentation for `Socket Runtime` can be found on our [Website](https://sockets.sh/)  
+The `Socket Runtime` documentation discusses API, examples , multiple guides (`Apple`,`Desktop`, and `Mobile`), `P2P` documentation and more.
```
Comment (chicoxyzzy) [inline README.md:46]: nit: extra space before the second comma

**Comment by chicoxyzzy** [comment]
I really love how the new Readme looks like

https://github.com/socketsupply/socket/blob/cfe8c839259eb4ba9cf3234289e901d9d02d36ab/README.md

**Comment by chicoxyzzy** [comment]
CI shields? Hmm... we have some of them failing. Maybe we can add them once we'll have all green?

**Comment by lamiazar** [comment]
> CI shields? Hmm... we have some of them failing. Maybe we can add them once we'll have all green?

Sure, sounds good.

File: README.md (hunk 3 above)
Comment (heapwolf) [reply README.md:15]: I think we could improve it slightly with this language:

> Web Developers use Socket Runtime to create apps for any OS, desktop or mobile. You can use plain old HTML, CSS, and JavaScript, as well as your favorite front-end libraries for example React, Svelt, and Vue.
>
> Socket Runtime exposes primitives needed for building peer-to-peer and local-first applications, such as Bluetooth, UDP, and robust file system access. Our P2P component can help you connect your app's users, and let them communicate directly, without the cloud or any servers at all.
>
> The Socket's CLI compiles applications into hybrid-native applications &mbasp; meaning, a combination of web code running in a platform's "WebView" along with platform-native code: Java on Android, Swift/Objective-C on iOS, C++ on Windows or Linux, etc.

**Review by heapwolf — CHANGES_REQUESTED** [review]
Perhaps consider my improvements to the intro wording, but otherwise this looks great.

**Comment by lamiazar** [comment]
@heapwolf I just updated the introduction.

File: README.md [hunk 6]
```diff
@@ -1,23 +1,64 @@
...
 
-Socket is a client-side runtime for creating native cross-platform software on
-mobile and desktop using HTML, CSS, and JavaScript. It also exposes primitives
-needed for building peer-to-peer and local-first applications, such as Bluetooth,
-UDP and File I/O.
+Web Developers use `Socket Runtime` to create apps for any OS, desktop, or mobile. You can use plain old HTML, CSS, and JavaScript, as well as your favorite front-end libraries for example React, Svelt, and Vue.
+
+`Socket Runtime` exposes primitives needed for building peer-to-peer and local-first applications, such as Bluetooth, UDP, and robust file system access. Our P2P component can help you connect your app's users, and let them communicate directly, without the cloud or any servers at all.
+
+The Socket's CLI compiles applications into hybrid-native applications &mbasp; meaning, a combination of web code running in a platform's "WebView" along with platform-native code: Java on Android, Swift/Objective-C on iOS, C++ on Windows or Linux, etc.
```
Comment (chicoxyzzy) [inline README.md:19]:
```suggestion
The Socket's CLI compiles applications into hybrid-native applications &mbasp; meaning, a combination of web code running in a platform's "WebView" along with platform-native code: Kotlin/Java on Android, Swift/Objective-C on iOS, C++ on Windows or Linux, etc.
```

File: README.md [hunk 7]
```diff
@@ -1,23 +1,64 @@
...
+
+### 💡 Features
+
+* Local First, a full featured File system API & Bluetooth.
+* P2P & Cloud, built to support a new generation of apps that can connect directly to each other by providing a high performance UDP API.
+* Use any backend, business logic can be written in any language, Python, Rust, Node.js, etc. The backend is even completely optional.
+* Use any frontend, you can use your favorite front end framework to create your UIs, React, Svelte, Vue.
```
Comment (chicoxyzzy) [inline README.md:26]:
```suggestion
* Use any frontend, you can use your favorite front end framework to create your UIs: React, Svelte, Vue.
```

File: README.md [hunk 8]
```diff
@@ -1,23 +1,64 @@
...
+
+This is similar to React's `Create React App`, we provides a few basic boilerplates and some strong opinions so you can get coding on a production-quality app as quickly as possible.  
+Please check [create-socket-app Repo](https://github.com/socketsupply/create-socket-app) to get started and to learn more.  
+You can also check our `Examples` in the [Examples Repo](https://github.com/socketsupply/socket-examples).  
+
+
+### 📚 Documentation
+
+The full documentation for `Socket Runtime` can be found on our [Website](https://sockets.sh/)  
+The `Socket Runtime` documentation discusses API, examples , multiple guides(`Apple`,`Desktop`, and `Mobile`),`P2P` documentation and more.
```
Comment (chicoxyzzy) [inline README.md:46]:
```suggestion
The `Socket Runtime` documentation covers Socket APIs, includes examples, multiple guides (`Apple`,`Desktop`, and `Mobile`), `P2P` documentation and more.
```

File: README.md [hunk 9]
```diff
@@ -1,23 +1,64 @@
...
+
+
+### 🧪 Testing
 
-### Documentation
+Test all the things! See the [`test/`](test/) for the unit and integration test suite.
```
Comment (chicoxyzzy) [inline README.md:51]: We can add that Socket provides a built-in test runner similar to `node:test` which outputs the test results in [TAP](https://testanything.org/) format

File: README.md (hunk 6 above)
Comment (lamiazar) [reply README.md:19]: @chicoxyzzy for the description/intro was suggested by @heapwolf so I did the update, I'll update the readme with your changes.

**Review by chicoxyzzy — APPROVED** [review]

File: README.md (hunk 6 above)
Comment (heapwolf) [reply README.md]: > The Socket's CLI
should be `The Socket Runtime CLI`

Comment (lamiazar) [reply README.md]: I will make the change now

**Review by jwerle — APPROVED** [review]

**Review by heapwolf — APPROVED** [review]

**Pull request merged**